website_portal_medical_patient/models/medical_patient.py

```diff
@@ -18,3 +18,13 @@ def action_invalidate(self):
     def _compute_website_url(self):
         for record in self:
             record.website_url = "/medical/patients/%s" % (record.id)
+
+    @api.model
+    def _search_related_patients(self):
```

Comment:
Minor detail, but if this method is being called from outside the model (in this case, from a controller), it shouldn't be "private" (i.e. have a name starting with an underscore).